tests/managed_services/mas_debezium/conftest.py

```diff
@@ -0,0 +1,199 @@
+import logging
+
+import pytest
+import rhoas_kafka_instance_sdk
+from constants import KAFKA_NAME, KAFKA_TOPICS, TEST_RECORD, TEST_TOPIC
+from ocp_resources.namespace import Namespace
+from ocp_resources.pod import Pod
+from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
+from ocp_utilities.infra import cluster_resource
+from rhoas_kafka_instance_sdk.api import acls_api, records_api, topics_api
+from rhoas_kafka_instance_sdk.model.acl_binding import AclBinding
+from rhoas_kafka_instance_sdk.model.acl_operation import AclOperation
+from rhoas_kafka_instance_sdk.model.acl_pattern_type import AclPatternType
+from rhoas_kafka_instance_sdk.model.acl_permission_type import AclPermissionType
+from rhoas_kafka_instance_sdk.model.acl_resource_type import AclResourceType
+from rhoas_kafka_instance_sdk.model.config_entry import ConfigEntry
+from rhoas_kafka_instance_sdk.model.new_topic_input import NewTopicInput
+from rhoas_kafka_instance_sdk.model.record import Record
+from rhoas_kafka_instance_sdk.model.topic_settings import TopicSettings
+from rhoas_kafka_mgmt_sdk.model.kafka_request_payload import KafkaRequestPayload
+from rhoas_service_accounts_mgmt_sdk.model.service_account_create_request_data import (
+    ServiceAccountCreateRequestData,
+)
+
+from utilities.template_utils import render_template_from_dict
+
+
+LOGGER = logging.getLogger(__name__)
+WAIT_STATUS_TIMEOUT = 120
+
+
+@pytest.fixture(scope="class")
+def kafka_instance(kafka_mgmt_api_instance):
+    _async = True
+    kafka_request_payload = KafkaRequestPayload(
+        cloud_provider="aws",
+        name=KAFKA_NAME,
+        region="us-east-1",
+        plan="standard.x1",
+        reauthentication_enabled=True,
+    )
+    kafka_create_api = kafka_mgmt_api_instance.create_kafka(
```

Comment:
does `create_kafka` have logs? if not, please relevant info log (relevant in all places where an action is taken)

Reply:
`create_kafka` does not produce any logs, currently we have logs for sampling the instance's status during creation time
using the `TimeoutSampler` as well as logging info regarding the instance once it's ready.
not sure I understand `all places where an action is taken`

Reply:
Please a log in each of the fixture with the action that s being done (e.g here - create kafka {name} instance...)

Reply:
done